docs/content/en/blog/articles/why-k8ssandra/index.md

```diff
@@ -0,0 +1,70 @@
+
+## Integration with the Rest of Your Stack
+
+K8ssandra is designed to support common infrastructure that you’re likely already using in your cloud-based deployments, instead of requiring separate infrastructure specific to Cassandra. 
+
+For example, let’s consider observability, and specifically metrics. K8ssandra deploys the DataStax Metrics Collector for Apache Cassandra in the same Kubernetes pod alongside each Cassandra node, to extract metrics and push them to Prometheus. Grafana is configured as a visualization tool for these metrics, including dashboards that allow you to monitor the key Cassandra and OS metrics indicating the health of each node. This makes it a simple task to create integrated views showing application and database metrics side-by-side, for a holistic view of system performance and health.
+
+Another example is how K8ssandra leverages TCP [ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/) for exposing Cassandra’s native binary protocol, allowing microservices or other application code running outside the Kubernetes cluster to access data using the Cassandra Query Language (CQL). The default configuration leverages [Traefik](/docs/topics/ingress/traefik/), but that may be replaced with the ingress implementation of your choice to expose CQL access for your environment.
+
+## Optimizing Developer Productivity, Cost, and Performance
+
+While fast access to CQL endpoints is attractive to developers who are already using Cassandra, most developers we’ve talked with would prefer to code to data APIs for new development, rather than learning a new database query language. This is why K8ssandra also includes [K8ssandra also includes Stargate](/docs/topics/stargate/), a data services gateway, which provides REST, Document, and GraphQL APIs on top of Cassandra. These familiar APIs lead to increased productivity and reduced time to market.
+
+Because Stargate nodes are Cassandra-compatible, they participate in the Cassandra cluster. This architecture confers additional benefits: Stargate nodes do the compute intensive co-ordination, while the Cassandra nodes handle data storage. This means that you can configure Cassandra deployments in Kubernetes with the right mix of compute-intensive Stargate and storage-intensive Cassandra machines for your use case. By selecting different instance types for Stargate and Cassandra nodes, you can achieve the ideal balance of cost and performance for your deployment.
```

Comment:
Added a hyphen and removed another:

```suggestion
Because Stargate nodes are Cassandra-compatible, they participate in the Cassandra cluster. This architecture confers additional benefits: Stargate nodes do the compute-intensive coordination, while the Cassandra nodes handle data storage. This means that you can configure Cassandra deployments in Kubernetes with the right mix of compute-intensive Stargate and storage-intensive Cassandra machines for your use case. By selecting different instance types for Stargate and Cassandra nodes, you can achieve the ideal balance of cost and performance for your deployment.
```